src/dcat-ap/dcat-dataset.ts

```diff
@@ -44,9 +44,11 @@ export function getDcatDataset(hubDataset: any, orgBaseUrl: string, orgTitle: st
   } as DatasetResource);
   const { relative: relativePath } = getContentSiteUrls(content, siteModel);
   const landingPage = siteUrl.startsWith('https://') ? siteUrl + relativePath : `https://${siteUrl}${relativePath}`;
-  
+  const downloadLink = siteUrl.startsWith('https://') ? `${siteUrl}/datasets/${hubDataset.slug}` : `https://${siteUrl}/datasets/${hubDataset.slug}`;
```

Comment:
I would change this to use `hubDataset.identifier`. That will return the slug if available (and the hubId on the off chance it's not)

Reply:
Done!